<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::editor::PickPlaceOutputJobWidget</class>
 <widget class="QWidget" name="librepcb::editor::PickPlaceOutputJobWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CSV pick&amp;place position file export for boards.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtOutputTop"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Boards:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rbtnBoardsAll">
         <property name="text">
          <string>All</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpBoards</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnBoardsDefault">
         <property name="text">
          <string>Default</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpBoards</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnBoardsCustom">
         <property name="text">
          <string>Custom:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpBoards</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lstBoards">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Assembly Variants:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rbtnVariantsAll">
         <property name="text">
          <string>All</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpVariants</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnVariantsDefault">
         <property name="text">
          <string>Default</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpVariants</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnVariantsCustom">
         <property name="text">
          <string>Custom:</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnGrpVariants</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lstVariants">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cbxCreateTop">
     <property name="text">
      <string>Output Top:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cbxCreateBottom">
     <property name="text">
      <string>Output Bottom:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cbxCreateBoth">
     <property name="text">
      <string>Output Combined:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtOutputBottom"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="edtOutputBoth"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="cbxIncludeComment">
     <property name="text">
      <string>Include metadata as comments</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Technologies:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbxTechnologyTht">
       <property name="text">
        <string>THT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTechnologySmt">
       <property name="text">
        <string>SMT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTechnologyMixed">
       <property name="text">
        <string>Mixed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTechnologyFiducial">
       <property name="text">
        <string>Fiducial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxTechnologyOther">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>cbxTechnologyTht</tabstop>
  <tabstop>cbxTechnologySmt</tabstop>
  <tabstop>cbxTechnologyMixed</tabstop>
  <tabstop>cbxTechnologyFiducial</tabstop>
  <tabstop>cbxTechnologyOther</tabstop>
  <tabstop>cbxCreateTop</tabstop>
  <tabstop>edtOutputTop</tabstop>
  <tabstop>cbxCreateBottom</tabstop>
  <tabstop>edtOutputBottom</tabstop>
  <tabstop>cbxCreateBoth</tabstop>
  <tabstop>edtOutputBoth</tabstop>
  <tabstop>cbxIncludeComment</tabstop>
  <tabstop>rbtnBoardsAll</tabstop>
  <tabstop>rbtnBoardsDefault</tabstop>
  <tabstop>rbtnBoardsCustom</tabstop>
  <tabstop>lstBoards</tabstop>
  <tabstop>rbtnVariantsDefault</tabstop>
  <tabstop>rbtnVariantsAll</tabstop>
  <tabstop>rbtnVariantsCustom</tabstop>
  <tabstop>lstVariants</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnGrpBoards"/>
  <buttongroup name="btnGrpVariants"/>
 </buttongroups>
</ui>
